v0/17-10 Creating_report_templates.markdown

```diff
@@ -0,0 +1,35 @@
+# 17.1.1    Creating {{report}} templates
```

Comment:
Isn't this 17.10.0 - I think the filename might need to be 17-10-0 too, I'm not totally sure but think it might be.